spec/factories/user.rb

```diff
@@ -1,7 +1,14 @@
 # This will guess the User class
 FactoryGirl.define do
   factory :user do
-    email {Faker::Internet.email}
-    password "password123"
+    email 'user@example.com'
+    password 'password'
+    admin false
   end
-end
\ No newline at end of file
+
+  factory :admin, class: User do
+    email 'admin@example.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.